rdmo/management/assets/js/utils/elements.js

```diff
@@ -174,4 +174,24 @@ const buildUri = (element) => {
   return uri
 }
 
-export { compareElements, updateElement, resetElement, canMoveElement, moveElement, findDescendants, buildUri }
+const buildPath = (key, parentUri, elements) => {
+
+  let path = key
+  if (parentUri) {
+    const parentElement = elements.find(element => element.uri === parentUri)
```

Comment:
I've added this find `path` from another parent element but I'm not sure if it's needed here to build the path only for attributes.

Would only this `parentPath = parentUri.split('/domain/')[1]` not be sufficient to build the path for a single attribute element?

Reply:
Yes, looking at this function, and if its only needed for Attributes, then I would simplify the function like you described.